src/config/configFile.ts

```diff
@@ -182,14 +182,16 @@ export class ConfigFile<T extends ConfigFile.Options> extends BaseConfigStore<T>
   /**
    * Used to initialize asynchronous components.
    *
-   * **Throws** *`Error`{ code: 'ENOENT' }* If the username.json file is not found when
+   * **Throws** *`Error`{ code: 'ENOENT' }* If the {@link ConfigFile.getFilename} file is not found when
    * options.throwOnNotFound is true.
    */
   protected async init(): Promise<void> {
```

Comment:
This is how I was proposing we fix the issue @shetzel - lets chat about it in the morning.

Reply:
Yes, this is essentially how I fixed it locally.  It all works this way, and is backward compatible, but being static makes it a less intuitive ux imo.  For example, the link in the comment isn't exactly accurate since it's not `ConfigFile.getFilename` but `this.constructor.getFilename`.  Seems more intuitive to just be `this.getFilename`, as well as `this.getDefaultOptions`.